Main.qml

```diff
@@ -5,16 +5,19 @@ import QtQuick.Controls.Material 2.12
 import QtQuick.Window 2.12
 import QtQuick.Controls.Universal 2.12
 import QtQuick.Layouts 1.12
+import QtQml 2.12
 import Qt.labs.platform 1.1
+import Qt.labs.settings 1.0
 
 ApplicationWindow {
     id: window
     visible: true
+    visibility: settings.windowVisibility
```

Comment:
Are you sure we want to save this? When you launch a program, it should always show, even if it was minimized on exit

Reply:
This is actually about maximized/windowed state. Maybe it's better to just save an isMaximized bool.

Reply:
That makes sense.